### PR TITLE
Fix LVM-not-detected log rendering

### PR DESCRIPTION
**Pull request by copejon**
Oneliner to make the warning log line render the error message.

### CODE REVIEW

File: pkg/components/storage.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func startCSIPlugin(cfg *config.MicroshiftConfig, kubeconfigPath string) error {
 	)
 
 	if err := lvmd.LvmSupported(); err != nil {
-		klog.Warning("skipping CSI deployment: %v", err)
+		klog.Warningf("skipping CSI deployment: %v", err)
```
Comment (eggfoobar) [inline pkg/components/storage.go:75]: I'm not 100% on how klog handles error types, but I think we should change the `%v` to `%w`. The go team has a nice little FAQ on some of these conventions. [see](https://github.com/golang/go/wiki/ErrorValueFAQ#i-am-already-using-fmterrorf-with-v-or-s-to-provide-context-for-an-error-when-should-i-switch-to-w)

**Review by pacevedom — COMMENTED** [review]
/lgtm

**Pull request merged**